vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -304,6 +310,62 @@ void FrameworkElementViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate,
         }
         AnnounceLiveRegionChangedIfNeeded(element);
       }
+      else if (propertyName == "accessibilityRole")
+      {
+        if (propertyValue.isString())
+        {
+          const std::string& role = propertyValue.getString();
+          if (role == "none")
```

Comment:
Looks like this name to enum could be described with a map. Not sure how often this is _really_ going to get run so don't know how much the hash will help. But... could be cleaner?

Reply:
Yeah, we have a lot of cleanup work in this codebase for all of these string compares. But it might be something that waits for fabric, when we have to replace the viewmanagers anyway.